packaging/supervisord/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM centos:7
+
+ARG rpm_file=cloudify-manager-install.rpm
+ARG config=config.yaml
+ARG uid=1500
+ARG gid=1500
+ARG reporter_uid=1501
+ARG reporter_gid=1501
+
+RUN groupadd -g $gid cfyuser
+RUN useradd -ms /bin/bash -g $gid --home-dir /etc/cloudify -u $uid cfyuser
+
+# This should save some time for the installations of all these programs
+RUN yum install -y openssl-1.0.2k libselinux-utils \
+    logrotate python-setuptools python-backports \
+    python-backports-ssl_match_hostname which cronie \
+    systemd-sysv initscripts tcp_wrappers-libs sudo openssh-server
+RUN easy_install supervisor
+
+EXPOSE 80 443 53333
```

Comment:
No rabbitmq port exposed?

Reply:
Yeah, rabbit port should be there, will add it 
```
EXPOSE 80 443 5672 53333
```

Reply:
5671- TLS. We don't use 5672 any more.